src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneService.java

```diff
@@ -0,0 +1,16 @@
+package uk.ac.ebi.atlas.experimentpage.markergenes;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+public interface MarkerGeneService {
```

Comment:
This is my view to use interfaces:
- Interfaces give more flexibility in the application. Of course, this is a generic statement.
- Interfaces avoid tight coupling in the application. We are injecting concrete classes where ever we required, this way makes our classes tightly coupled with each other. If we introduce an interface, we can inject(DI) an interface instead of a concrete class. Then we can operate on interfaces instead of implementation. 
- Interfaces act as an abstraction for our implementation classes.
- If we use interfaces will end up with fewer code changes. As we are injecting(DI)only interfaces, so it will not rely on the implementation of class changes.
I have mentioned a few advantages whatever I remembered.